### PR TITLE
Clean up CYA pages

### PR DESCRIPTION
**Pull request by lara-mpt**
### Change description ###
Responding to comments on https://github.com/hmcts/nfdiv-frontend/pull/1557

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/steps/respondent/check-your-answers/content.ts [hunk 1]
```diff
@@ -248,7 +248,7 @@ const languages = {
 
 export const generateContent: TranslationFn = content => {
   const translations = languages[content.language](content);
-  const applicant2Url = content.isApplicant2 ? urls.RESPONDENT : '';
+  const applicant2Url = urls.RESPONDENT;
```
Comment (velimd) [inline src/main/steps/respondent/check-your-answers/content.ts:251]: wouldnt this break where its being used, as itll be underfined

Comment (lara-mpt) [reply src/main/steps/respondent/check-your-answers/content.ts:251]: when would it be undefined?

Comment (lara-mpt) [reply src/main/steps/respondent/check-your-answers/content.ts:251]: https://github.com/hmcts/nfdiv-frontend/pull/1557#discussion_r772215279

Comment (velimd) [reply src/main/steps/respondent/check-your-answers/content.ts:251]: yep, youre right, ignore me, i was thinking if it was checking if its app2

**Review by velimd — APPROVED** [review]

**Pull request merged**